Common subdirectories: cl-rsm-string-1.3.orig/debian and cl-rsm-string-1.3/debian
Only in cl-rsm-string-1.3: package.x86f
Only in cl-rsm-string-1.3: semantic.cache
diff -u cl-rsm-string-1.3.orig/string.lisp cl-rsm-string-1.3/string.lisp
--- cl-rsm-string-1.3.orig/string.lisp	2003-10-22 21:35:29.000000000 -0500
+++ cl-rsm-string-1.3/string.lisp	2004-08-06 23:02:18.627475008 -0500
@@ -75,8 +75,7 @@
                      (concatenate 'string s1 join-string s2)) str-list))))
 
 
-(declaim (ftype (function ((list string) (list string) 
-                                         &key function) list)
+(declaim (ftype (function ((or list string) (or list string) &key (:function function)) list)
                 contains))
 
 (defun contains (str-list elem-list &key (contain-meaning #'some))
@@ -92,9 +91,6 @@
       collect str))
 
 
-(declaim (ftype (function (list string &key (:unique t)) list)
-                does-not-contain))
-
 (defun does-not-contain (str-list str-elems &key (unique nil))
   "Return a list of strings from the string list, <str-list>, that do not
 contain any of the strings from the list of strings <str-elems.> If unique is
Only in cl-rsm-string-1.3: string.lisp~
